sql/mob_spell_lists.sql

```diff
@@ -1504,6 +1504,429 @@ INSERT INTO `mob_spell_lists` VALUES ('Lord_of_Onzozo',152,201,1,255); -- Waterg
+INSERT INTO `mob_spell_lists` VALUES ('Megamaw_Mikey',180,191,1,255); -- Stonega III
+INSERT INTO `mob_spell_lists` VALUES ('Megamaw_Mikey',180,210,1,255); -- Quake
+
+-- Piasa
+INSERT INTO `mob_spell_lists` VALUES ('Piasa',181,102,1,255); -- Enaero
+
+-- Poroggo_Dom_Juan
+INSERT INTO `mob_spell_lists` VALUES ('Poroggo_Dom_Juan',182,201,1,255); -- Waterga III
+INSERT INTO `mob_spell_lists` VALUES ('Poroggo_Dom_Juan',182,226,1,255); -- Poisonga II 
+
+-- Abas
+INSERT INTO `mob_spell_lists` VALUES ('Abas',183,172,1,255); -- Water IV
+INSERT INTO `mob_spell_lists` VALUES ('Abas',183,201,1,255); -- Waterga III
+INSERT INTO `mob_spell_lists` VALUES ('Abas',183,214,1,255); -- Flood
+INSERT INTO `mob_spell_lists` VALUES ('Abas',183,226,1,255); -- Poisonga II
+
+-- Cannered_Noz
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,152,1,255); -- Blizzard IV
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,153,1,255); -- Blizzard V
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,181,1,255); -- Blizzaga III
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,182,1,255); -- Blizzaga IV
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,206,1,255); -- Freeze
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,207,1,255); -- Freeze II
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,274,1,255); -- Sleepga II
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,356,1,255); -- Paralyga
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,359,1,255); -- Silencega
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,366,1,255); -- Graviga
+INSERT INTO `mob_spell_lists` VALUES ('Cannered_Noz',184,367,1,255); -- Death
+
+-- Chukwa
+INSERT INTO `mob_spell_lists` VALUES ('Chukwa',185,192,1,255); -- Stonega IV
+INSERT INTO `mob_spell_lists` VALUES ('Chukwa',185,202,1,255); -- Waterga IV
+INSERT INTO `mob_spell_lists` VALUES ('Chukwa',185,211,1,255); -- Quake II
+INSERT INTO `mob_spell_lists` VALUES ('Chukwa',185,215,1,255); -- Flood II
+
+-- Cuelebre
+INSERT INTO `mob_spell_lists` VALUES ('Cuelebre',186,357,1,255); -- Slowga
+INSERT INTO `mob_spell_lists` VALUES ('Cuelebre',186,365,1,255); -- Breakga
+INSERT INTO `mob_spell_lists` VALUES ('Cuelebre',186,54,1,255);  -- Stoneskin 
+
+-- Glavoid
+INSERT INTO `mob_spell_lists` VALUES ('Glavoid',187,192,1,255); -- Stonega IV
+INSERT INTO `mob_spell_lists` VALUES ('Glavoid',187,211,1,255); -- Quake II
+INSERT INTO `mob_spell_lists` VALUES ('Glavoid',187,357,1,255); -- Slowga
+
+-- Lachrymater
+INSERT INTO `mob_spell_lists` VALUES ('Lachrymater_Fireday',188,147,1,255); -- Fire IV
+INSERT INTO `mob_spell_lists` VALUES ('Lachrymater_Fireday',188,176,1,255); -- Firaga III
+INSERT INTO `mob_spell_lists` VALUES ('Lachrymater_Fireday',188,204,1,255); -- Flare
+INSERT INTO `mob_spell_lists` VALUES ('Lachrymater_Fireday',188,235,1,255); -- Burn
+INSERT INTO `mob_spell_lists` VALUES ('Lachrymater_Fireday',188,249,1,255); -- Blaze spikes
+
```

Comment:
@R3P0FFXI have a look at https://github.com/DarkstarProject/darkstar/blob/824fbbd7c79eea6d43524c0d96131abffac56881/scripts/zones/Lower_Delkfutts_Tower/mobs/Disaster_Idol.lua like I said last yesterday. Without a script he'll be locked into his starting element's list.